Integrations/integration-opswat-metadefender.yml

```diff
@@ -0,0 +1,239 @@
+        var ec = {};
+        var dbotScore = 0;
+
+        md += '### Results for scan id ' + args.id + '\n';
+
+        if (res.file_info) {
+            if (res.process_info.progress_percentage < 100) {
+                md += '### The scan proccess is in progrees (done: ' + res.process_info.progress_percentage + '%) \n'
+            }
+            md += 'File name: ' + res.file_info.display_name + '\n';
+            md += 'File description: ' + res.file_info.file_type_description + '\n';
+            md += 'Scan result: ' + res.scan_results.scan_all_result_a + '\n';
+            md += 'Detected AV: ' + res.scan_results.scan_all_result_i + '/' + res.scan_results.total_avs + '\n';
+            md += 'Scan progress percentage ' + res.process_info.progress_percentage + '\n';
+
+            var percntBad = (res.scan_results.scan_all_result_i / res.scan_results.total_avs) * 100;
+
+            ec['OPSWAT(val.ScanId == ' + args.id + ').ScanProgress'] = res.process_info.progress_percentage;
+            ec['OPSWAT(val.ScanId == ' + args.id + ').PercentageBad'] = percntBad;
+            ec['OPSWAT(val.ScanId == ' + args.id + ').Result'] = res.scan_results.scan_all_result_a;
+
+
+            md += 'AV Name|Def Time|Threat Name Found\n';
+            md += '---|---|---\n';
+
+            var avRes = res.scan_results.scan_details;
+
+            Object.keys(avRes).forEach(function(key) {
+              md += key + '|';
+              md += avRes[key].def_time + '|';
+              md += avRes[key].threat_found + '\n';
+            });
+
+
+
+            if (percntBad >= 66) {
+                dbotScore = 3;
+                addMalicious(ec, outputPaths.file,{
+                    Hash: res.file_info.md5,
+                    Malicious: {Vendor: 'OPSWAT', Description: 'Result from OPSWAT-Metadefender'}
+                });
+            } else if (percntBad >= 34) {
+                dbotScore = 2;
+            } else {
+                dbotScore = 1;
+            }
+            ec.DBotScore = {Indicator: res.file_info.md5, Type: 'hash', Vendor: 'OPSWAT', Score: dbotScore};
+
+
+            return {Type: entryTypes.note, Contents: res, ContentsFormat: formats.json, HumanReadable: md, EntryContext: ec};
+        } else {
+            md += 'No results for this id\n';
+
+            return {Type: entryTypes.note, Contents: res, ContentsFormat: formats.json, HumanReadable: md, EntryContext: ec};
+        }
+    };
+
+
+    switch (command) {
+        // This is the call made when pressing the integration test button.
+        case 'test-module':
+            var res = sendRequest('/hash/', 'BED12FDA073BB386B54700138FB47EEA');
+            return 'ok';
+        case 'opswat-hash':
+            return searchHash();
+        case 'opswat-scan-file':
+            return scanFile();
+        case 'opswat-scan-result':
+            return getScanResult();
+        default:
+            throw 'Unknown command - ' + command;
+    }
+  type: javascript
+  commands:
```

Comment:
commands and arguments descriptions are missing

Reply:
Added